### PR TITLE
Add welcome page 7

### PR DESCRIPTION
**Pull request by Errolyn**
Code pr: https://github.com/mozilla/bedrock/pull/8671

Lang Checker: https://github.com/mozilla-l10n/langchecker/pull/969

Note: Eric will be adding French and German translations

### CODE REVIEW

File: en-US/firefox/welcome/page7.lang [hunk 1]
```diff
@@ -0,0 +1,75 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/welcome/7/
```
Comment (peiying2) [inline en-US/firefox/welcome/page7.lang]: The link doesn't work

Comment (Errolyn) [reply en-US/firefox/welcome/page7.lang]: The link will work in just a bit :D we haven't push to dev yet but it will be there today.

Comment (peiying2) [reply en-US/firefox/welcome/page7.lang]: The langchecker PR is stuck with Travis. I can't merge it.

Comment (peiying2) [reply en-US/firefox/welcome/page7.lang]: The URL still doesn't work this morning.

**Pull request merged**